gopowerstore_types.go

```diff
@@ -67,7 +67,7 @@ func (err *APIError) NotFound() bool {
 
 // VolumeNameIsAlreadyUse returns true if API error indicate that volume name is already in use
 func (err *APIError) VolumeNameIsAlreadyUse() bool {
-	return err.StatusCode == http.StatusUnprocessableEntity
+	return err.StatusCode == http.StatusUnprocessableEntity || err.StatusCode == http.StatusInternalServerError
```

Comment:
For all cases, can we assume Internal server error corresponds to volume already in use state only?

Reply:
This error code is a generic one thrown from API and hence works for all the cases without any issues.